ethereum/permissioning/src/main/java/org/hyperledger/besu/ethereum/permissioning/TransactionSmartContractPermissioningController.java

```diff
@@ -194,7 +196,8 @@ private static Bytes encodeTransaction(final Transaction transaction) {
         encodeAddress(transaction.getSender()),
         encodeAddress(transaction.getTo()),
         transaction.getValue().toBytes(),
-        transaction.getGasPrice().toBytes(),
+        // TODO: no treatment of 1559 fields here, is Bytes32.ZERO an appropriate fallback?
```

Comment:
Same question as to the other TODO. Did we forget this here or will we deal with this later?

Reply:
added to #2441, thanks again for catching